Change who should get bug reports. We don't want Ruby folks to be bothered with our mistakes.

diff --git a/test/debugger-ext/test-000-error.sh b/test/debugger-ext/test-000-error.sh
new file mode 100755
index 0000000..9b9f6f1
--- /dev/null
+++ b/test/debugger-ext/test-000-error.sh
@@ -0,0 +1,2 @@
+#!/bin/bash
+(cd ../.. && ./miniruby -I./lib -I. -I.ext/common  ./tool/runruby.rb --extout=.ext  -- --disable-gems  ./test/ruby/test_rubyoptions.rb)
diff --git a/test/ruby/test_rubyoptions.rb b/test/ruby/test_rubyoptions.rb
index b6be501..7dfb88c 100644
--- a/test/ruby/test_rubyoptions.rb
+++ b/test/ruby/test_rubyoptions.rb
@@ -526,12 +526,6 @@ class TestRubyOptions < Test::Unit::TestCase
       (?m:.*)
       \[NOTE\]\n
       You\smay\shave\sencountered\sa\sbug\sin\sthe\sRuby\sinterpreter\sor\sextension\slibraries.\n
-      Bug\sreports\sare\swelcome.\n
-      (?:.*\n)?
-      For\sdetails:\shttp:\/\/.*\.ruby-lang\.org/.*\n
-      \n
-      (?:#{additional})
-      \z
       )x
   end
